### PR TITLE
Perform upload error handling.

### PR DESCRIPTION
**Pull request by kenyavs**
Append body ticket error to upload exception string.

### CODE REVIEW

File: src/Vimeo/Vimeo.php [hunk 1]
```diff
@@ -347,7 +347,11 @@ public function replace($video_uri, $file_path, $upgrade_to_1080 = false, $machi
     private function perform_upload($file_path, $ticket)
     {
         if ($ticket['status'] != 201) {
-            throw new VimeoUploadException('Unable to get an upload ticket.');
+            if(isset($ticket['body']['error'])){
+		$ticket_error = "[" . $ticket['body']['error'] . "]";
+	    }
+
+ 	    throw new VimeoUploadException('Unable to get an upload ticket.' . $ticket_error);
```
Comment (Dashron) [inline src/Vimeo/Vimeo.php]: What if $ticket_error is undefined? I think it needs to be initialized at line 350 with an empty string.

Comment (kenyavs) [inline src/Vimeo/Vimeo.php]: Yes, I tried that case. Initially I had it initialized to an empty string but removed it when I tried it without initializing it and noticed that 'Null' or 'undefined' wasn't appended to the string.

I'll reinstate it for readability's sake...

Comment (kenyavs) [inline src/Vimeo/Vimeo.php]: ...never mind :) isset() only checks for NULL not undefined :/

Comment (Dashron) [inline src/Vimeo/Vimeo.php]: I don't think undefined is a thing in PHP, isset or empty should both work fine!

Comment (kenyavs) [inline src/Vimeo/Vimeo.php]: ha! I just added a check for it...

Comment (kenyavs) [inline src/Vimeo/Vimeo.php]: ...so when you said undefined you meant empty?

**Comment by vinkla** [comment]
@Dashron does the API always return an error for request other than `201`?

**Comment by vinkla** [comment]
@kenyavs When you're done with this pull request, could you please [squash your commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html).

**Comment by kenyavs** [comment]
@vinkla for sure, I'll squash them.

@Dashron suggested to use [] in order to identify the system error, error returned within the body. I'll switch it back to isset but isset won't catch '', empty will. I believe this was the concern...

**Comment by vinkla** [comment]
Okay. Do the API every return an empty message?

**Comment by kenyavs** [comment]
I think it's possible, yes because I wasn't checking for it...or rather I thought that isset would. But then @Dashron brought it to my attention.

**Pull request closed without merging**